sktime/forecasting/compose/_ensemble.py

```diff
@@ -87,16 +87,18 @@ def _predict(self, fh, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
         -------
         DataFrame : aggregate dataframe reduction.
         """
+        valid_agg_funcs = ("median", "mean", "min", "max")
+        d_frame = pd.concat(self._predict_forecasters(fh, X), axis=1)
```

Comment:
I would call `d_frame` `y_pred` instead or `Y_pred` if you want to indicate that it's not just a single prediction.